CHANGES.md

```diff
@@ -4,6 +4,8 @@
 
 ### _Black_
 
+- Fixed bug that Black doesn't do anything when
+  .gitignore file includes non-alphabetical(non-ascii) characters (#2229)
```

Comment:
```suggestion
- Fix handling of .gitignore files containing non-ASCII characters (#2229)
```
"Alphabetical" doesn't make sense; surely it was also broken if you wrote your gitignore in Cyrillic.